src/plugins/management/server/capabilities_provider.ts

```diff
@@ -17,6 +17,8 @@ export const capabilitiesProvider = () => ({
       indexPatterns: true,
       objects: true,
       aiAssistantManagementSelection: true,
+      aiAssistantManagementObservability: true,
+      aiAssistantManagementSecurity: true,
       observabilityAiAssistantManagement: true,
```

Comment:
Is there a corresponding `securityAiAssistantManagement` that should be here as well, or is `observabilityAiAssistantManagement` deprecated in favor of the two `aiAssistantManagement*` added above here?